### PR TITLE
formatted files with prettier

### PR DESCRIPTION
**Pull request by Lookout00**
updated files with prettier

### CODE REVIEW

File: aboutus.html [hunk 1]
```diff
@@ -1,77 +1,100 @@
...
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>About Us</title>
-</head>
```
Comment (trooperandz) [inline aboutus.html:8]: This is great.  I'd like to see more of this.

File: stylesheets/style_footer.css [hunk 2]
```diff
@@ -1,136 +1,135 @@
...
-    text-transform: uppercase;
-    margin-top: 1px;
-    letter-spacing: 2px;
+  color: white;
+  font-size: 20px;
+  text-transform: uppercase;
+  margin-top: 1px;
+  letter-spacing: 2px;
 }
 /*BASSHOLES HYPERLINK*/
```
Comment (trooperandz) [inline stylesheets/style_footer.css:22]: Is this a throwback to 'Assholes'?

File: script_login.js
Comment (trooperandz) [inline script_login.js:1]: Did you write this code?

**Review by trooperandz — CHANGES_REQUESTED** [review]
I'd really like to see these changes, basshole.

File: aboutus.html (hunk 1 above)
Comment (Lookout00) [reply aboutus.html:8]: Who's the basshole now? do you play cod?

**Pull request merged**